airflow/helm/airflow/values.yaml.tpl

```diff
@@ -4,8 +4,10 @@ global:
     - description: airbyte web ui
       url: {{ .Values.hostname }}
 
+{{ if .Values.redisEnabled }}
```

Comment:
Similar issue here, use {{ if not .Values.redisDisabled }} to turn this off manually but default-on.